cadg-ui/src/index.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import { Provider } from 'react-redux';
+import store from './store';
+import './css/index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <Provider store={store}>
```

Comment:
Why are you wrapping the store around the app here rather than in the app component? More curiosity than anything else, I thought you could do it in either place.

Reply:
The Provider is typically your top-most level component. Plus, once the Router is added, it will need to be a parent of the App component and a child of the Provider. I could technically put both of these within App.js, but putting it at the root index.js is cleaner imo so I can start creating routes immediately within the App component.

Reply:
Sounds good